### ISSUE TITLE
Sequences \x1b[0K and \x1b[2K not detected

### ISSUE THREAD

**Issue by jquast** [jquast/blessed#95]
From http://invisible-island.net/xterm/ctlseqs/ctlseqs.html
```
CSI Ps K  Erase in Line (EL).
            Ps = 0  -> Erase to Right (default).
            Ps = 1  -> Erase to Left.
            Ps = 2  -> Erase All.
```

Fail
```
In [7]: Sequence('\x1b[0K', blessed.Terminal()).length()
Out[7]: 3
In [9]: Sequence('\x1b[2K', blessed.Terminal()).length()
Out[9]: 3
```

Pass
```
In [8]: Sequence('\x1b[1K', blessed.Terminal()).length()
Out[8]: 0
In [10]: Sequence('\x1b[K', blessed.Terminal()).length()
Out[10]: 0
```

**Comment by jquast** [comment]
should just require additions to `CAPABILITY_DATABASE` in `blessed/_capabilities.py`

**Comment by jquast** [comment]
fixed in release `1.14.2`.
